### PR TITLE
Ignore retry.Config var

### PR DESCRIPTION
**Pull request by dvrkps**

### CODE REVIEW

File: builder/azure/arm/step_delete_resource_group.go [hunk 1]
```diff
@@ -108,6 +108,9 @@ func (s *StepDeleteResourceGroup) deleteDeploymentResources(ctx context.Context,
 			}
 			return err
 		})
+		if err != nil {
+			return err
```
[the remark this answers is not in the document]
Comment (dvrkps) [reply builder/azure/arm/step_delete_resource_group.go]: @azr @paulmey i am agains ignoring any errors but if you insists to  ignore this error maybe is better to log this error somehow ?

[the remark this answers is not in the document]
Comment (azr) [reply builder/azure/arm/step_delete_resource_group.go]: I think the error is reported line 107

[the remark this answers is not in the document]
Comment (dvrkps) [reply builder/azure/arm/step_delete_resource_group.go]: @azr i rename pr for better info.

retry.Config is struct and check if not nil is allways true.
this is wrong. i was tricked by var name.
better name for retry.Config is ```config``` but this var is not use anywhere.

question of all question is how pass compile:
```
err := retry.Config{}
if err !=nil {
    return err
}
```

i return retry.Config instead error in this function and compile not fails ?

[the remark this answers is not in the document]
Comment (paulmey) [reply builder/azure/arm/step_delete_resource_group.go]: Sorry for being late, but it looks like you've got it all figured out.
The error is indeed logged on line 107 and should not terminate the loop through all `deploymentOperations`. Capturing it in a variable was most likely a copy/paste artifact.
If we want to make that more explicit, we can not capture it at all, so remove the `_ = ` on line 98 and `return nil` on line 109?

File: builder/azure/arm/step_delete_resource_group.go [hunk 2]
```diff
@@ -95,7 +95,7 @@ func (s *StepDeleteResourceGroup) deleteDeploymentResources(ctx context.Context,
 			resourceType,
 			resourceName))
 
-		err := retry.Config{
+		_ = retry.Config{
```
Comment (paulmey) [inline builder/azure/arm/step_delete_resource_group.go]:
```suggestion
		retry.Config{
```

Comment (paulmey) [reply builder/azure/arm/step_delete_resource_group.go]: as well as `return nil` on line 109. Can't figure out how to suggest that on GitHub... 😉

Comment (dvrkps) [reply builder/azure/arm/step_delete_resource_group.go]: @paulmey @azr Done.

**Comment by azr** [comment]
LGTM ! Thanks ! 🙂

**Pull request merged**